### PR TITLE
DOCS-9653 Update Access Management docs

### PR DESCRIPTION
**Pull request by chris-campbell9**
JIRA:
https://www.mulesoft.org/jira/browse/DOCS-9653

### CODE REVIEW

File: modules/ROOT/pages/managing-api-clients.adoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ Before you delete a client provider, ensure that you do not have APIs, assets, o
 
 . Log in to Anypoint Platform using an account that has the organization administrator role.
 . In Anypoint Platform, click *Access Management*.
+. Under Business Groups, select your Master organization.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-api-clients.adoc]:
```suggestion
. Under *Business Groups*, select your master organization.
```

**Review by sarathecoubian — DISMISSED** [review]
Minor change.

**Comment by chris-campbell9** [comment]
@sarathecoubian Thanks for the review! Somehow I "dismissed" your approval when I committed your suggestion. Would you kindly re-approve? (I'll figure out what happened so this doesn't happen again).

**Review by sarathecoubian — DISMISSED** [review]
Dismissing my approval when making changes is behavior that always happens. No way to prevent it! Reapproved :)

**Review by franciscoSoler — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/environments.adoc [hunk 2]
```diff
@@ -90,14 +90,14 @@ You can now configure client providers for APIs.
...
-When you remove your default client provider from an environment that has more than one client provider, the next client provider is automatically named as your new default client provider.
+You can remove a client provider from an environment at any time. However, prior to removing the default client provider, specify another client provider as default.
+
 When you remove all external client providers from an environment, the built-in Anypoint Platform client provider becomes the default client provider for that environment.
 
 . Navigate to *Access Management*.
 . In the menu on the left, click *Environments*.
 . Click the name of the environment.
-. Click the *x* next to the name of the client provider that you want to remove.
+. Click the *Remove* button next to the name of the client provider that you want to remove. If the client provider is default, first specify another client provider as default by clicking *Set as default*.
```
Comment (priyaayyar) [inline modules/ROOT/pages/environments.adoc]:
```suggestion
. Click the *Remove* button next to the name of the client provider that you want to remove.
+
If the client provider is the default, first specify another client provider as default by clicking *Set as default*.
```

**Review by priyaayyar — DISMISSED** [review]

File: modules/ROOT/pages/environments.adoc [hunk 3]
```diff
@@ -90,14 +90,16 @@ You can now configure client providers for APIs.
...
+
 When you remove all external client providers from an environment, the built-in Anypoint Platform client provider becomes the default client provider for that environment.
 
 . Navigate to *Access Management*.
 . In the menu on the left, click *Environments*.
 . Click the name of the environment.
-. Click the *x* next to the name of the client provider that you want to remove.
+. Click the *Remove* button next to the name of the client provider that you want to remove.
++
+If the client provider is the default, first specify another client provider as default by clicking *Set as default*.
```
Comment (kbostrom) [inline modules/ROOT/pages/environments.adoc]: This possibly puts the steps in the wrong order, so I suggest instead...:
```suggestion
. If the client provider that you want to remove is the default, click *Set as default* next to the name of a different client provider.
+
. Click the *Remove* button next to the name of the client provider that you want to remove.

**Review by kbostrom — DISMISSED** [review]
CE per request

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**